### ISSUE TITLE
campaign table name hard set to 'campaign' in latest release (docs point to `latest`, not `stable`)

### ISSUE THREAD

**Issue by pamop** [NYUCCL/psiTurk#540]
just noticed an issue with the tagged release where the table names for hits and campaigns aren't being grabbed from the config.txt like they should be. it seems like this is handled correctly in the master branch where the tables are named appropriately at the beginning of the models.py file:

```
ASSIGNMENTS_TABLENAME = config.get('Database Parameters', 'table_name')
HITS_TABLENAME = config.get('Database Parameters', 'hits_table_name')
CAMPAIGNS_TABLENAME = config.get('Database Parameters', 'campaigns_table_name')
```

but in the 3.2.1 release, in the hit class
`__tablename__ = 'amt_hit'`

and in the campaign class
`__tablename__ = 'campaign'`

I've changed it locally for my own psiturk install (so i can run campaigns at the same time as other people with whom i share a database) but should be fixed so people who pip install the latest version don't end up with overlapping database tables!

**Comment by deargle** [comment]
Yikes! That's a problem with readthedocs pointing to the master branch instead of the most recent release. That feature has been sitting unreleased for many months now, but since it's on master, the docs suggest it's available.

**Comment by deargle** [comment]
I updated readthedocs to point to `stable` (the latest released version) instead of `latest` (the master branch)

**Comment by deargle** [comment]
Feature released in v3.3.0 https://github.com/NYUCCL/psiTurk/releases/tag/v3.3.0
